core/roboconf-dm/src/main/java/net/roboconf/dm/management/api/ITargetsMngr.java

```diff
@@ -324,4 +324,14 @@ void dissociateTargetFrom( AbstractApplication app, String instancePathOrCompone
 	 * @return a non-null list of usage item
 	 */
 	List<TargetUsageItem> findUsageStatistics( String targetId );
+
+
+	/**
+	 * Finds the script that will be executed once the VM is created.
+	 * @param targetId a target ID
+	 * @return a non-null map (key = the file location, relative to the instance's directory, value = file content)
+	 * @throws IOException if something went wrong while reading a file
+	 */
+	Map<String,byte[]> findScriptResources( String targetId ) throws IOException;
```

Comment:
Good. Add a second one that takes an application name and a scoped instance path as arguments.